docs/sighup.md

```diff
@@ -10,8 +10,6 @@ Available for reconfiguration fields:
 
 ```yml
   head_timeout:
-  cache_size:
```

Comment:
a little bit confused why this happened in this PR. you just noted that it is outdated? a separate commit then?